### DISCUSSION TITLE
Online operators is still flickering in the latest version 4.39

### DISCUSSION THREAD

**Discussion by praveenvall** [LiveHelperChat/livehelperchat#2101] [General]
I have upgraded yesterday to 4.38 and found this and was about to report. 
But, suddenly I found a new version 4.49 and have upgraded to it. 
However, we are still having the same issue. The online operators is still flickering( which means sometimes it will disappear and upon touching the screen, it may come in a while).

And your mentions under the category are:
Online visitors attribute filter sometimes was lost.
Online operators tab was migrated to Svelte also.

I would like to know if I need to do anything on Svelte?

**Comment by remdex** [comment]
Record a video, you are the only who has this issue

**Comment by fabioascoelho** [comment]
Hi @remdex 
I’m also experiencing an issue since I updated my LHC from version 4.29 to 4.45. The Online Operators Panel is flickering, and every time it refreshes, it displays a different status for the online operators. When I recorded the video [Online Operators Flicker.zip](https://github.com/user-attachments/files/16637870/Online.Operators.Flicker.zip), there were 8 operators online consistently, but the panel showed incorrect status throughout.


Best regards, 
Fábio

**Reply by remdex** [reply]
if official demo does not have that issue i can't help. For that is paid support. Most of the times, users just mess up with things they don't understand and then they come saying that it does not work :D

Also make sure you have value something like this.
![image](https://github.com/user-attachments/assets/32d85846-600a-476b-84cd-2ef5cb84fed9)

**Reply by fabioascoelho** [reply]
Thanks! That worked. I had a 5-second value in that option

**Reply by Onpraveen** [reply]
> I had a 5-second value in

@fabioascoelho 
hello,
can you please tell me which value was showing you 5 seconds because even i had this issue on my old instance.

Kind regards

**Reply by remdex** [reply]
This config value never had 5 as default value. It was always 300 which is 5 minutes...

**Reply by fabioascoelho** [reply]
> > I had a 5-second value in
> 
> @fabioascoelho hello, can you please tell me which value was showing you 5 seconds because even i had this issue on my old instance.
> 
> Kind regards
Hi, sure. I’m sorry for the delay. 
It's just bellow
https://github.com/orgs/LiveHelperChat/discussions/2101#discussioncomment-10439221

**Comment by praveenvall** [comment]
Fabio,

Could you please elaborate how did you find the solution .
Point me to that specific file and like or a screenshot if possible .


Praveen Valluri
+91-962 081 2323(What's App)
+91-911 362 6157
***@***.***

**Reply by fabioascoelho** [reply]
Hi, sure. I’m sorry for the delay.

The setting is located under Home / System Configuration / Synchronization and Sound Settings:
![image](https://github.com/user-attachments/assets/d3dd1d5c-6855-47c3-9262-a89efa80ee00)

It is found on the first panel of the Settings, just below Geo Location under the System title.

The first option is “How many seconds for a user to be considered as being online.” Mine was set to 5 seconds on my production server. I don’t know why, because on the backup and homologation servers, it was set to 200 seconds. Either someone changed this setting without my knowledge, or it was altered during the update from version 4.29 to 4.45.

**Reply by Onpraveen** [reply]
excellent
